x-pack/test/security_solution_cypress/ai4dsoc_serverless_config.ts

```diff
@@ -34,6 +34,8 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
         ])}`,
         '--csp.strict=false',
         '--csp.warnLegacyBrowsers=false',
+        '--xpack.features.overrides.securitySolutionTimeline.hidden=true',
+        '--xpack.features.overrides.securitySolutionNotes.hidden=true',
```

Comment:
Do we need to force this? Should it be read from the config automatically when using this `productTypes`? 🤔

Reply:
@semd Yes for now the way the tests are setup, cypress doesn't consider the tier specific "override" yml files that we are now using. This was the simplest way to add that in the test setup. But I agree we should make this better so it reads from the config files instead.

Reply:
@ashokaditya Are you sure? I don't see all the regular `serverless.security.yml` configs being duplicated in the `x-pack/test/security_solution_cypress/serverless_config.ts`. 
They might be a reason why this is not working for the tier configs, but I don't think we should duplicate the entire tier yaml here.

@MadameSheema could you help?

Reply:
@MadameSheema @semd I'm going to add test changes in a subsequent PR.